source/SH3/graphics/texture.cpp

```diff
@@ -23,12 +23,21 @@
 
 using namespace sh3_graphics;
 
-#pragma pack(push, 1)
+/** @defgroup graphics-types Graphics Types
+ *  @{
+*/
+
+/**
+ *  RGBA pixel
+ */
 struct rgba
```

Comment:
These types should be wrapped in an anonymous namespace to avoid collision with the types in `color.hpp`.
I'm uncertain about the name resolution then though. I think with just `rgba` you'd get the packed version and with `::rgba` you can get the unpacked one.
Still it might be better to call them `packed_*` or something to avoid confusion.

Reply:
Yeah I think that would look a lot nicer than `::rgba` imho.